snap/snapcraft.yaml

```diff
@@ -12,22 +12,13 @@ grade: devel
 confinement: strict
 
 apps:
-  run: 
-    command: server
+  cli: 
```

Comment:
not sure I understand. do you suggest "wifi-connect.wif-connect" instead of "wifit-connect.cli"?. If so, that just feels redundant to me  (and is longer).